Makefile

```diff
@@ -134,39 +141,33 @@ $(OBJ_DIR)/%.o: %.c
 	@mkdir -p $(dir $@)
 	$(CC) $(_CFLAGS) $< -o $@ -c
 
-$(OBJ_DIR)/%.cov.o: %.c
+$(OBJ_DIR)/%.dbg.o: %.c
 	@mkdir -p $(dir $@)
-	$(CC) $(_CFLAGS) -c --coverage $< -o $@
+	$(CC) $(_CFLAGS) -c --coverage -g $(SANITIZE_FLAGS) $< -o $@
 
 $(BIN_DIR)/secvarctl: $(OBJS) $(LIBSTB_SECVAR)
 	@mkdir -p $(BIN_DIR)
 	$(CC) $(_CFLAGS) $(STATICFLAG) $^ -o $@ $(_LDFLAGS)
 
-$(BIN_DIR)/secvarctl-cov: $(OBJCOV) $(LIBSTB_SECVAR)
+$(BIN_DIR)/secvarctl-dbg: $(OBJDBG) $(LIBSTB_SECVAR)
 	@mkdir -p $(BIN_DIR)
-	$(CC) $(_CFLAGS) $^ $(STATICFLAG) -fprofile-arcs -ftest-coverage -o $@ $(_LDFLAGS)
+	$(CC) $(_CFLAGS) $^ $(STATICFLAG) -g $(SANITIZE_FLAGS) -fprofile-arcs -ftest-coverage -o $@ $(_LDFLAGS)
 
 $(LIBSTB_SECVAR):
 	$(MAKE) CFLAGS=-DSECVAR_CRYPTO_WRITE_FUNC -C external/libstb-secvar lib/libstb-secvar-openssl.a
 
 
 secvarctl: $(BIN_DIR)/secvarctl
-secvarctl-cov: $(BIN_DIR)/secvarctl-cov
-.PHONY: secvarctl secvarctl-cov
+secvarctl-dbg: $(BIN_DIR)/secvarctl-dbg
+debug: $(BIN_DIR)/secvarctl-dbg
+.PHONY: secvarctl secvarctl-dbg debug
 
-check: secvarctl-cov
+check: secvarctl-dbg
```

Comment:
we are going to need to update documentation at some point. README and man file will probably need a new review.

Reply:
Indeed. I'll overhaul the build / testing instructions after the build system changes stabilize.